lib/parser.js

```diff
@@ -35,15 +35,26 @@ async function parse(asyncapiYAMLorJSON, options = {}) {
     throw new ParserError(e.message);
   }
 
+  const getData = async url => {
+    try {
+      const response = await fetch(url);
+      const json = await response.json();
+      return json;
+    } catch (error) {
+      console.log(error);
+    }
+  };
+
   const ajv = new Ajv({
     allErrors: true,
+    schemaId: 'auto',
   });
+  ajv.addMetaSchema(await getData('http://json-schema.org/draft-04/schema'));
```

Comment:
Can you provide a full example of when this is failing? I can't manage to make it fail.

Reply:
Also, we should not go to the Internet to grab the schema. Ajv already got this packaged we just have to do `ajv.addMetaSchema(require('ajv/lib/refs/json-schema-draft-04.json'));`. Otherwise, the parser will not work when offline.

Reply:
Oh! I just saw it failing on Travis.

Reply:
It's already fixed on the base branch. Thanks for raising the issue! I guess this PR can be closed now.